producer/client.js

```diff
@@ -2,28 +2,74 @@
 var express = require("express");
 var http = require('http');
 var amqp = require('amqplib/callback_api');
+var mongodb = require('mongodb');
 var myParser = require("body-parser");
 var app = express();
 var args = process.argv.slice(2);
+var employee;
 
-var employee = {
-    FirstName: "John",
-    LastName: "Snow",
-    Position: "None",
-    Salary: 0,
-    Location: "Winterfell",
-    MaritalStatus: "Single",
-    LastUpdated: new Date(2010, 01, 01),
-    Version: 0,
+
+var Mango = {
```

Comment:
Hehe

Reply:
Thought it fit in with the spirit of RabbitWrangler